src/odemis/acq/stream/__init__.py

```diff
@@ -40,7 +40,7 @@
 if sys.version_info >= (3, 4):
     ABC = abc.ABC
 else:
-    ABC = abc.ABCMeta('ABC', (object,), {'__slots__': ()})
+    ABC = abc.ABCMeta(str('ABC'), (object,), {'__slots__': ()})
```

Comment:
`'ABC'` == `str('ABC')`, right? So what's the difference?

Reply:
You're right, it doesn't change anything. In a different commit that I haven't pushed yet I import unicode_literals in every file in odemis. I'm not sure if that's a good idea. On the one hand it makes sense to be consistent across the entire codebase (we do need to import it in some files), but on the other hand it might also introduce a lot of unintended bugs.

What do you think?

If unicode_literals is imported in this file, we need to do this conversion to string here.

Reply:
Actually, originally, I had in mind that we would add an "import unicode_literals" to every file which we have tested both with Python 2 and 3... however, I hadn't foreseen such issue, where external code behaves differently with str and unicode (on Python 2). As shown here, using the unicode_literals forces  us to write ugly code, which wouldn't be necessary otherwise.   

So maybe we should adjust this idea, and say: if possible, use "import unicode_literals", but if that makes the code too ugly, just test on python 2 & 3, and add a comment just before the `import __future__` to indicate the code has been tested, but unicode_literals will break it.

Here, I'd suggest to drop this change.